git-session.html

```diff
@@ -115,6 +116,22 @@
            }
            slides[slideIndex-1].style.display = "block";  
          }
+
+         //Added by Harsha to post a feed.
+         function addFeed() {
+            var feedTextArea = document.getElementById("feedTextArea");
+            var feedsList = document.getElementById("feedsList");
+            var feedToUpdate = feedTextArea.value;
+            if("" != feedToUpdate && undefined != feedToUpdate && null != feedToUpdate){
```

Comment:
No need to check != for empty string, undefined and null values
As in a condition they will always give false

Reply:
https://j11y.io/javascript/truthy-falsey/